pkg/customresources/keymanagementproviders/kmpcertmanager.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright The Ratify Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package keymanagementproviders
+
+import (
+	"crypto/x509"
+	"sync"
+
+	"github.com/deislabs/ratify/internal/constants"
+	kmp "github.com/deislabs/ratify/pkg/keymanagementprovider"
+)
+
+// ActiveCertStores implements the KMPCertManager interface.
+type ActiveCertStores struct {
+	// scopedStores maps from scope to CertificateMap defined in /pkg/keymanagementprovider/keymanagementprovider.go
+	// Example:
```

Comment:
The KMP Certificate map already has the namespace prepended to the name as a key. Is it necessary to have scopedStores?